source/style-guide/writing/write-to-the-user.txt

```diff
@@ -82,11 +113,14 @@ talk to users directly by using *second person*, addressing the user as
 .. _subjunctive-mood:
 
 Avoid the Subjunctive Mood
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+--------------------------
 
 The subjunctive mood expresses doubt. *That doesn’t engender trust from
-readers about our content!* Avoid words like “should” and “could”; they
-imply uncertainty.
+readers about our content!* Avoid words that imply uncertainty:
+
```

Comment:
I would add, to help drive home:

Readers expect the documentation to be authoritative. If you find yourself in a situation where you cannot reliable know which of two options is the correct one, research with engineering or the team to find the exact answer and document it! Don't use language that leaves the discrepancy up to the user to ponder!

Reply:
Ah found one reasonable counter. Such words are appropriate for our production notes, as they are performance recommendations and guidelines. "You should configure your vm.swappiness to 1 if possible"

This doc: https://docs.mongodb.com/manual/administration/production-notes/

"Must: or similar would be inappropriate, because the site sysadmin knows the best value for their deployment: we, instead, are providing recommendations that should be enacted if possible, but can be ignored if required.